runners/cromwell_on_google/wdl_runner/file_util.py

```diff
@@ -15,7 +15,7 @@
 
 from googleapiclient import discovery
 from googleapiclient.errors import HttpError
-from oauth2client.client import GoogleCredentials
+#from oauth2client.client import GoogleCredentials
```

Comment:
Why drop the credentials?

Reply:
we deprecated oauth2client in favor of google-auth in our python libraries so its not needed

Reply:
Awesome, let's remove it entirely then